### ISSUE TITLE
Top left logo on v2 should probably go to www

### ISSUE THREAD

**Issue by pgleeson** [OpenSourceBrain/OSBv2#770]
So that there is some clear way to go back to the new homepage from any page on osbv2...

**Comment by filippomc** [comment]
Not sure if exiting the application is really what the user expects clicking there.. what if we add the link somewhere else instead?

**Comment by pgleeson** [comment]
True, but on Github or Google docs & others if you click the top left logo you go to the homepage and leave the current file/doc even if you're editing it. Seems to be a standard workflow...

**Comment by pgleeson** [comment]
A better option might be to put a link to "Home" in the Dashboard menu, or rename the Support so Home is appropriate here...

![Screenshot 2023-10-18 at 11 11 22](https://github.com/OpenSourceBrain/OSBv2/assets/1556687/954c37cc-ea1f-47b0-88ac-4d558276903c)

**Comment by pgleeson** [comment]
Done
